src/screens/TopicPageScreen/index.js

```diff
@@ -120,6 +122,10 @@ const TopicPageScreen = (props) => {
         setIsInitialLoading(false);
       }
     } catch (error) {
+      SimpleToast.show(
+        'There has been a problem. Please make sure you’re online. ',
+        SimpleToast.SHORT
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job adding error handling and user feedback when an error occurs. However, it might be beneficial to provide more specific error messages based on the type or content of the error, rather than a generic message for all errors.
```diff
- SimpleToast.show(
-   'There has been a problem. Please make sure you’re online. ',
-   SimpleToast.SHORT
- );
+ SimpleToast.show(
+   error.message || 'There has been a problem. Please make sure you’re online.',
+   SimpleToast.SHORT
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->